### ISSUE TITLE
[Bug(s)]  further investigations into multiplayer malfunctions

### ISSUE THREAD

**Issue by Battlepixel** [tsunamayo/Starship-EVO#3573]
### Short Description of issues:
This time:
My friend (client) was able to join the game, but started again in a dark environment (no stars, ships just several characters nestling into one another) in which he could not move. In this version he could teleport to me via the teleportation function. He could only move there in "no-clipping mode" and not interact with the environment. He saw me open doors and hangar gates, but he didn't see me. I didn't see him either. Objects that I spawned were also invisible to him. After 5 minutes, I couldn't interact with the environment either and we tried a second time. On the second try he could see me, but no ships, objects or stars. Furthermore, he could only move in "no-clipping mode".

### Steps to reproduce the issue:
Try to play Multiplayer

### Environment
- Starship EVO version: 21w05a
- Graphics Card: Host: gtx 1070 (mobile), Client gtx 1050TI (mobile)

### Additional information:
[PlayerLog (Host).zip](https://github.com/tsunamayo/Starship-EVO/files/5926721/PlayerLog.Host.zip)
[PlayerLog (Client).zip](https://github.com/tsunamayo/Starship-EVO/files/5926760/PlayerLog.Client.zip)

**Comment by StarrKiss** [comment]
Yeah, this has happened for me too for quite a few versions back, exactly as you describe here.

**Comment by tsunamayo** [comment]
I tested it on my side (had to fix a few bugs) before the release - so I dont reproduce that... How far are you guys located? Also when you say you quit and retry again, did you quit to windows (it is better to do so). I will see if I can reproduce, it is complicated to test...
@StarrKiss I fixed a few issues that must have been there since a couple version back. This one should be better, if you have the opportunity to test... (if it works or not, ping me ;))

**Comment by Battlepixel** [comment]
Yes we always quit to windows.

It doesn't seem to matter how far away we are or whether we start a ne world.

When the client joins it starts in a gray parallel world without objects and the possibility to interact. When he teleports out, he can still only move in "no-clip mode", but he sees all objects that were already there when he joined and also changes such as opening doors or gates. The client cannot interact, it only sees what the host is doing. However, the host and client cannot see each other. When we restarted the game and server, the client did not spawn in the same location where it left the game. He could only move in f5 mode again and there were no objects for him, but this time he could see me and I him.

We had the same problem before the patch, only that the teleporting out didn't work and the join only in one of 5 attempts. Joinen worked again without errors this time, but not everything else.

**Comment by tsunamayo** [comment]
@Battlepixel okaaay I finally managed to reproduce, I was testing with Build as Host and Unity Editor as client, and it worked. If I use two build I got the error that you faced. I use the editor so I can check both log, other I can only get the client log. Anyway issue is fixed for 21w05b. I am now fixing an issue where the client cant spawn a new entity and will push the build... 
Thanks for your patience...
